docs/core/setup/install-runtime.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Install .NET Core runtime on Windows, Linux, and macOS - .NET Core
+description: Learn how to install .NET Core on Windows, Linux, and macOS. Discover the dependencies required to run .NET Core apps.
+author: thraka
+ms.author: adegeo
+ms.date: 11/06/2019
+ms.custom: "updateeachrelease"
+zone_pivot_groups: operating-systems-set-one
+---
+
+# Install the .NET Core Runtime
+
+In this article, you'll learn how to download and install the .NET Core runtime. The .NET Core runtime is used to run apps created with .NET Core.
```

Comment:
Is it also called the common language runtime, and should we mention that?

Reply:
No, in this case it's the product they are downloading. The actual runtime used is called `CoreCLR` (to differentiate it from the .NET Framework CLR) However, I do need to capitalize these references to runtime since it's the product name.